t/t3428-rebase-edit-todo.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+test_description='rebase should reread the todo file if an exec modifies it'
+
+. ./test-lib.sh
+
+todo=$(git rev-parse --git-dir)/rebase-merge/git-rebase-todo
```

Comment:
--git-dir in this particular instance indeed returns .git/. so we can get away with hard-coding it.

In case of submodules or worktrees, this is not the case, but in the test suite, we know that we do not have such a beast, as we implicitly initialize a regular repository.

You are correct about the spaces, of course. I was actually thinking of arbitrary white space (which gets converted into spaces unless quoted).